ctools/defragment_sharded_collection.py

```diff
@@ -821,7 +823,7 @@ async def phase_2():
 
         with tqdm(total=total_chunks_to_process, unit=' chunks') as progress:
             iteration = 0
-            while iteration < 25:
+            while True:
```

Comment:
I slightly prefer having a variable here telling me the condition for leaving the loop - by this way, it becomes easier to understand where and why we are done.

Reply:
Unfortunately python doesn't have the do/while construct, so I would need to introduce somenthing like:
```
while not in first_round and condition_still_meet:
   loop
```